### PR TITLE
QUICKSTEP-79 Fixed headers and NOTICE/LICENSE files

### PR DESCRIPTION
**Pull request by hbdeshmukh**

### CODE REVIEW

**Comment by cramja** [comment]
LTGM, covers what Josh said in the @dev chat. There's still more to do in the NOTICE/LICENSE file, however.

**Comment by hbdeshmukh** [comment]
I rebased my fork as well. Do you want to merge this now or wait for the NOTICE/LICENSE updates?

**Comment by cramja** [comment]
I am working on the license file now- I sent out a question to Josh about which clauses to include. How about we wait on this PR and I'll have the changes for LICENSE soon and we can merge them.

**Comment by hbdeshmukh** [comment]
Sounds good to me.

**Comment by hbdeshmukh** [comment]
Hi @cramja I cherry picked the commit from your branch and I am going to merge this PR now, as I already got your approval.

File: NOTICE [hunk 1]
```diff
@@ -5,33 +5,4 @@ This product includes software developed at
...
-
-[Copyright for third_party/src/farmhash]
-Copyright (c) 2014 Google, Inc.
-
-[Copyright for third_party/src/gflags]
-Copyright (c) 2006, Google Inc.
-
-[Copyright for third_party/src/glog]
-Copyright (c) 2008, Google Inc.
+Portions Copyright (c) 2015-2016, Pivotal Software, Inc.
```
Comment (zuyu) [inline NOTICE]: This line missed `EOL`.

**Pull request merged**